average.js

```diff
@@ -1,6 +1,7 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  const newArray = numbers.filter( value => !Number.isNaN(value) );
```

Comment:
Try a better name than `newArray` to let readers know the intent of that variable without having to read the entire code.